src/qibo/backends/__init__.py

```diff
@@ -27,30 +26,36 @@ def load(backend: str, **kwargs) -> Backend:
 
         Args:
             backend (str): Name of the backend to load.
-            kwargs (dict): Additional arguments for the qibo backend.
+            kwargs (dict): Additional arguments for the ``qibo`` backend.
+
         Returns:
-            qibo.backends.abstract.Backend: The loaded backend.
+            :class:`qibo.backends.abstract.Backend`: Loaded backend.
         """
 
-        if backend == "numpy":
-            return NumpyBackend()
-        elif backend == "tensorflow":
-            return TensorflowBackend()
-        elif backend == "pytorch":
+        possible_backends = ["numpy", "pytorch", "clifford", "qulacs"]
+
+        if backend not in possible_backends:
+            raise_error(
+                ValueError,
+                f"Backend {backend} is not available. "
+                + f"The native qibo backends are {QIBO_NATIVE_BACKENDS}.",
+            )
+
+        if backend == "pytorch":
             return PyTorchBackend()
-        elif backend == "clifford":
+
+        if backend == "clifford":
             engine = kwargs.pop("platform", None)
             kwargs["engine"] = engine
+
             return CliffordBackend(**kwargs)
-        elif backend == "qulacs":
-            from qibo.backends.qulacs import QulacsBackend
+
+        if backend == "qulacs":
+            from qibo.backends.qulacs import QulacsBackend  # pylint: disable=C0415
 
             return QulacsBackend()
-        else:
-            raise_error(
-                ValueError,
-                f"Backend {backend} is not available. The native qibo backends are {QIBO_NATIVE_BACKENDS}.",
-            )
+
+        return NumpyBackend()
```

Comment:
I am not sure I like this new form of the `load`, I think we are giving up a little bit of readability, and I am not sure why we needed to change this.

Reply:
The `load` function is not ending in a return, which is a `lint` issue. I don't know how much any else care about this, so if y'all want the other way, at least the lint issue should be manually disabled then.